ros2/BUILD.bazel

```diff
@@ -21,6 +21,8 @@ exports_files([
     "test.bzl",
     "test.py.tpl",
     "topic.bzl",
+    "ament_setup.cpp.tpl",
+    "ament_setup.hpp.tpl",
```

Comment:
In this repo Google style guide is used. Hence, ament_setup.cc.tpl and ament_setup.h.tpl. Please alph-sort the list.